client-sdk/go/modules/contracts/oas20/oas20.go

```diff
@@ -0,0 +1,78 @@
+package oas20
+
+import (
+	"github.com/oasisprotocol/oasis-core/go/common/quantity"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/contracts"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+)
+
+// XXX: create a more highlevel helper wrapping Contracts client.
```

Comment:
Would be great to file an issue.

Reply:
Yeah i will in case it won't be done in this PR